qiita_ware/ebi.py

```diff
@@ -280,18 +280,20 @@ def generate_study_xml(self):
 
         return study_set
 
-    def add_sample(self, sample_name, taxon_id=None, description=None,
-                   **kwargs):
+    def add_sample(self, sample_name, taxon_id, scientific_name,
+                   description, **kwargs):
         """Adds sample information to the current submission
 
         Parameters
         ----------
         sample_name : str
             Unique identifier for the sample
-        taxon_id : str, optional
-            Defaults to ``None``. If not provided, the `empty_value` will be
-            used for the taxon ID
-        description : str, optional
+        taxon_id : str
```

Comment:
EBI submission is based on the template files, rather than on the data in the DB, and everything is parsed as a string.

We can refactor the EBI submission code to actually get the information from the DB, and I think this is something that we should do. In fact we already have an issue for it: https://github.com/biocore/qiita/issues/369. I would like to work on that one, so I assigned to myself to visit it after the ASM.